jdi-light-angular/pom.xml

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>jdi-light-angular</artifactId>
+    <name>JDI Angular</name>
+    <dependencies>
+        <dependency>
+            <groupId>com.epam.jdi</groupId>
```

Comment:
remove jdi-light while you already have jdi-light-html

Reply:
Thanks, fixed